src/main.rs

```diff
@@ -55,22 +55,28 @@ fn main() {
 
     match listener_res {
         Ok(mut listener) => {
-            let _ = load_complete(Ok(()), duration).or_else(|_| listener.close());
+            let _ = load_complete(Ok(()), duration, None, None).or_else(|_| listener.close());
         }
         Err(err) => {
             error!("{} {} Failed to load: {}", LOG_IDENTIFIER, "-", err);
-            let _ = load_complete(Err(err), duration);
+            let _ = match err {
+                Error::UnexpectedExit(code, stdout, stderr) => load_complete(
```

Comment:
```
match err {
    err @ Error::UnexpectedExit(code, stdout, stderr) => load_complete(Err(err.clone()), ...etc...),
    err => load_complete(Err(err), ...etc...),
}
```

but it's really not much worse than what you did an reiterates why the error handling in langserver is poorly designed

Reply:
i think i had tried `err.clone()` and got this:
```
err.clone()
   |                         ^^^^^ method not found in `langserver::error::Error`
```
so i just re-created the object :shrug:.  I'm doing a little more digging into just adding stderr directly to the `UnexpectedExit`'s `message` since i think that would end up waaaay simpler and I think accomplish the same goal

Reply:
checked into this again and Error doesn't have the #[derive CLone] and moreover can't have it because some of the subtypes don't have clone either:
```
error[E0277]: the trait bound `serde_json::Error: std::clone::Clone` is not satisfied                                                           
error[E0277]: the trait bound `hyper::client::Response: std::clone::Clone` is not satisfied
                                                                                                                                                      error[E0277]: the trait bound `std::io::Error: std::clone::Clone` is not satisfied                                                         
error[E0277]: the trait bound `hyper::Error: std::clone::Clone` is not satisfied                                                                      
```